requirements.txt

```diff
@@ -11,7 +11,7 @@ Jinja2==3.0.3
 jira>=2.0.0
 jsonschema>=3.0.2
 prison>=0.1.2
-prometheus_client>=0.10.1
```

Comment:
Same version the https://github.com/jertel/elastalert2 is using, but need to review the rest of the changes before I update from their master